.ci/code_generation.py

```diff
@@ -58,17 +61,22 @@
             os.remove(f)
     except:
         pass
-core.start_local_server(config=core.AvailableServerConfigs.LegacyGrpcServer)
+core.start_local_server(config=core.AvailableServerConfigs.GrpcServer)
 code_gen = core.Operator("python_generator")
```

Comment:
This will only work for PREMIUM. Should we explicitly add it as context?

Reply:
@rafacanton, sure, we can add it explicitely at the beginning of the script.
Also, sorry about the confusion Rafael, the problems with the constructors of the operators is that there is a second step in the operator generation, which I now added to the script.